index.html

```diff
@@ -283,50 +283,73 @@
 
 ## People &amp; Data {#people-data}
 
-A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>) is any natural
-person.
-
-We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=] such
-that:
-
-* this [=person=] is identified, directly or indirectly, by reference to an
-  identifier such as a name, email address, an arbitrary identifier or
-  identification number, an online identifier such as an IP address or any
-  identifier attached to a device this [=person=] may be using, phone number,
-  location data, or factors specific to the physical, physiological, genetic,
-  mental, economic, cultural, or social identity of that [=person=], as well as
-  identifiers derived from such data, for instance through hashing; or
-* this [=person=] could reasonably be reidentified from a conjunction of this
-  data with other data; or
-* the data pertains to a group of people such that a [=person=] may find
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>)
+is any natural person.
+
+We define <dfn data-lt="identifier">identifying data</dfn> as any information relating to a
+[=person=] such that any of the following applies:
+
+1. It references this [=person=] directly, for instance through their name,
+  email address, phone number, or other such traditional denotation.
+* It can identify this [=person=] through an arbitrary string or identification
+  number, an online identifier such as an IP address or any identifier attached to a
+  device which this [=person=] may be using.
+* It is about a [=person=]'s location (or of a device that a [=person=] would use) with greater
+  granularity than the city level or a comparably coarse geographic grouping.
+* It contains factors specific to the physical, physiological, genetic, mental,
+  economic, cultural, or social identity of a [=person=].
+* It is derived from other types of data in this list, for instance through hashing.
+* The [=person=] could reasonably be reidentified from a conjunction of this
+  [=data=] with other data.
+* It relates to a group of people such that a [=person=] may be treated in a
+  specific way for being part of the group, even if the entity treating them in this
+  manner has no way to identify that [=person=]. Put differently, an aggregate of [=people=]
+  is identifying if the aggregate is actionable with respect to its members.
+
+We define <dfn data-lt="data">personal data</dfn> as any information that either is
+[=identifying data=] or is related to [=identifying data=].
```

Comment:
This has been reworked significantly — please look at it again! @jyasskin @sandandsnow

Reply:
"Is related to identifying data" might be too broad ... or ambiguous: is "the speed limit of Main St" personal data because it happens to be where particular people are located and so related to some identifying data, or does the data have to actively "relate" the two by storing the connection between them?

Reply:
This is a thorny question (inferences and threat models for geo data). Knowing my location AND knowing the speed limit where I am is potentially problematic since you might be in possession of evidence of a misdemeanour!

I wonder if we shouldn't keep "relate" for now and make a separate issue about the varying threats of "relations"?